pool_bin

```diff
@@ -66,13 +67,19 @@ def gc(disable_recursion=False):
 
 def read_packages(in_file):
     packages = []
-    with open(in_file, "r") as fob:
-        for line in fob.readlines():
-            line = line.split("#")[0].strip()
-            if not line:
-                continue
-            packages.append(line)
-    return packages
+    try:
+        with open(in_file, "r") as fob:
+            for line in fob.readlines():
+                line = line.split("#")[0].strip()
+                if not line:
+                    continue
+                packages.append(line)
+        return packages
+    except FileNotFoundError as e:
+        if not DEBUG:
+            fatal(e)
+        else:
+            raise e
```

Comment:
Style note, pretty sure you can `raise` here without the `e` as you are re-raising the same exception.

Reply:
Cheers mate. Funny you should mention that. I actually noticed that when I did a final review myself but I had alreayd pushed to GH by then, so left it be. I might just fix that now, then push.